applications/nrf5340_audio/tools/buildprog/buildprog.py

```diff
@@ -224,7 +254,33 @@ def __main():
                         const=True,
                         help="Run nrfjprog sequentially instead of in \
                         parallel.")
+    #DFU relative option
+    parser.add_argument("-m", "--min_b0n", default=False,
+                        action='store_true',
+                        help="net core bootloader use minimal size build")
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument("-M", "--mcuboot", default=False,
+                        action='store_true',
+                        help="enable internal flash dfu")
+    group.add_argument("-e", "--external_flash", default=False,
+                        action='store_true',
+                        help="enable external flash dfu")
+
     options = parser.parse_args(args=sys.argv[1:])
+    global USE_MCUBOOT
```

Comment:
and here it might be nicer to use args.mcuboot instead of global variables. I would also pass these options to the function that uses them, instead of relying on a global parameter, but that is personal preference.

Reply:
Add options parameter to related functions.